tests/key_field/key_field_sample_generated.h

```diff
@@ -67,11 +67,12 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(1) Baz FLATBUFFERS_FINAL_CLASS {
   bool KeyCompareLessThan(const Baz * const o) const {
     return KeyCompareWithValue(o->a()) < 0;
   }
-  int KeyCompareWithValue(const flatbuffers::Array<uint8_t, 4> *_a) const { 
-    for (auto i = 0; i < a()->size(); i++) {
-      const auto a_l = a_[i];
+  int KeyCompareWithValue(const flatbuffers::Array<uint8_t, 4> *_a) const {
+    const auto* curr_a = a();
+    for (auto i = 0; i < curr_a->size(); i++) {
+      const auto a_l = curr_a->Get(i);
```

Comment:
I would just name these `lhs` and `rhs` and forgo adding the "name" part. I don't think you lose much and it will be smaller code.